includes/class-fs-logger.php

```diff
@@ -240,6 +251,9 @@ function api_error( $api_result, $wrapper = false ) {
 		}
 
 		function entrance( $message = '', $wrapper = false ) {
+            if ( ! $this->is_on() ) {
```

Comment:
This might not be needed since the `_log` is already bailing. If you want to be more memory/compute optimistic, then we need to have this check for every public log related function, but that isn't needed IMO.